OneSignalSDK/onesignal/src/main/java/com/onesignal/NotificationBundleProcessor.java

```diff
@@ -510,7 +510,7 @@ public void run() {
 
    // OSNotificationExtensionService still makes additional checks such as notValidOrDuplicated
    private static boolean startExtenderService(Context context, Bundle bundle, ProcessedBundleResult result) {
-       Intent intent = OSNotificationIntentService.getIntent(context);
+       Intent intent = new Intent(); //OSNotificationIntentService.getIntent(context);
```

Comment:
can this comment be deleted?

Reply:
No, this is something else not related to Android X migration